src/Microsoft.DotNet.XHarness.CLI/CommandArguments/Android/AndroidInstallCommandArguments.cs

```diff
@@ -44,15 +44,22 @@ protected override OptionSet GetTestCommandOptions() => new()
                 v => DeviceId = v
             },
             {
-                "boot-timeout=", "Timeout in seconds to wait for a device after boot completion",
-                v => {
-                    if (int.TryParse(v, out var number))
+                "launch-timeout=|lt=", "TimeSpan or number of seconds to wait for a device after boot competion",
```

Comment:
I would avoid "TimeSpan" and put something like Time span in the form of "00:00:00" or something like that.

Do we have it in iOS like this?

Reply:
yes 
https://github.com/dotnet/xharness/blob/7c3503d264dce638d85ef58278dd3d59bbb40ad4/src/Microsoft.DotNet.XHarness.CLI/CommandArguments/Apple/AppleTestCommandArguments.cs#L67
and for common timeout as well

Reply:
fixed evetywhere